fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/ExecFunction.java

```diff
@@ -43,4 +43,9 @@
      * return type
      */
     String returnType();
+
+    /**
+     * hasVarArgsc
+     */
+    boolean varArgs();
```

Comment:
```java
boolean varArgs() default false;
```

then you can skip to set varArgs to other functions

Reply:
good idea, done